src/extension.ts

```diff
@@ -1,16 +1,23 @@
 import { ExtensionContext } from 'vscode';
 
-import { registerURIHandler } from './lib/uri';
+import { getRootPath } from './lib/helpers';
 import { registerCommands } from './lib/commands';
+import { registerEvents } from './lib/events';
+import { registerURIHandler } from './lib/uri';
 import { initializeStatusBar } from './lib/statusBar';
-import { getRootPath } from './lib/helpers';
 import { initializeProviders } from './lib/providers';
 import { initializeConfig } from './lib/auth';
 import { initializeTerminal } from './lib/terminal';
-import { registerEvents } from './lib/events';
+import { initializeTracking, trackEvent } from './lib/tracking';
+import { TRACKED_EVENTS } from './lib/constants';
 
 export const activate = async (context: ExtensionContext) => {
-  console.log('Activating Extension...');
+  initializeTracking(context);
+  await trackEvent(TRACKED_EVENTS.ACTIVATE);
```

Comment:
Curious if it might make sense to track OS version here as well instead of adding a separate event for it? Seems like the best place for it 
https://git.hubteam.com/hubspot/cms-developer-tooling/issues/59

Reply:
Oh, looks like we track that for every event. A separate tracker may be redundant then 😄